docs/troubleshooting_guide.md

```diff
@@ -0,0 +1,358 @@
+```
+
+* Fix:
+Ensure that the entries for the n/w interfaces are correctly defined:
+```bash
+✔ ~/git-clone-holder/bms-toolkit [master L|✚ 114…15] 
+20:02 $ cat cluster_config.json
+20:03 $ cat cluster_config.json.enough_fun
+[
+  {
+    "scan_name": "dec-test-scan-cluster",
+    "scan_port": "1521",
+    "cluster_name": "dec-test-cluster",
+    "cluster_domain": "dec-test-home",
+    "public_net": "dec-test-public_interface",   <==================== this and 
+    "private_net": "dec-test-private",           <==================== this should reflect the actual interfaces on the BM hosts 
+    "scan_ip1": "172.16.30.20",
+    "scan_ip2": "172.16.30.21",
+    "scan_ip3": "172.16.30.22",
+    "dg_name": "DATA",
+    "nodes": [
+      {  "node_name": "racnode1.orcl",
+         "host_ip": "172.16.30.1",
+         "vip_name": "racnode1-vip.orcl",
+         "vip_ip": "172.16.30.11"
+      },
+      {  "node_name": "racnode2.orcl",
+         "host_ip": "172.16.30.2",
+         "vip_name": "racnode2-vip.orcl",
+         "vip_ip": "172.16.30.12"
+      }
+    ]
+  }
+]
+
+```
+
+The interface to be used for `public_net` is the interface that carries the network addrees for the `172.16.30.0` (in the example above) network.
+* Example:
+```bash
+[root@racnode1 ~]# ifconfig -a bond0.111
+bond0.111: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 172.16.30.1  netmask 255.255.255.0  broadcast 172.16.30.255
+        inet6 fe80::3680:dff:fe59:7f1e  prefixlen 64  scopeid 0x20<link>
+        ether 34:80:0d:59:7f:1e  txqueuelen 1000  (Ethernet)
+        RX packets 1277524  bytes 13573540106 (12.6 GiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 1227353  bytes 621025389 (592.2 MiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+The interface to be used for `private_net` is the one that carries the IP for the private interconnect which will be used exclusively for RAC traffic, for example `192.168.3.1` (in the example below):
+
+```bash
+[root@racnode1 ~]# ifconfig -a bond1.112
+bond1.112: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
+        inet 192.168.3.1  netmask 255.255.255.0  broadcast 192.168.3.255
+        inet6 fe80::3680:dff:fe59:7f1f  prefixlen 64  scopeid 0x20<link>
+        ether 34:80:0d:59:7f:1f  txqueuelen 1000  (Ethernet)
+        RX packets 5664  bytes 260540 (254.4 KiB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 5670  bytes 238460 (232.8 KiB)
+        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
+```
+
+#### 4. ASM candidate disks having a header containing old DG data
+
+* Error:
+```bash
+SEVERE:  [Jan 20, 2021 7:42:09 AM] [FATAL] [INS-30530] Following specified disks have invalid header status: 
+[/dev/asmdisks/DATA_7539397476, /dev/asmdisks/DATA_7539397477, /dev/asmdisks/DATA_7539397478, /dev/asmdisks/DATA_7539397531]
+```
+
```

Comment:
Perhaps a short explanation of what this means:  there is existing data on the disk, and we don't want to corrupt a disk.  Carefully review the disk (using a tool like less -f) and use cleanup-oreacle.sh to clear out a preivous install if necessary.

Reply:
Added details